testonly/hammer/hammer.go

```diff
@@ -453,11 +481,6 @@ func (s *hammerState) performInvalidOp(ctx context.Context, ep MapEntrypointName
 }
 
 func (s *hammerState) getLeaves(ctx context.Context) error {
-	const (
-		existingKey = iota
-		nonexistentKey
-		numOptions
-	)
 	which, ok := s.pickCopy()
```

Comment:
As above, prefer to keep the available choices visible locally, something like:
```go
func (s *hammerState) getLeaves(ctx context.Context) error {
	choices := []Choice{ExistingKey, NonExistentKey}
	...
	for i := 0; i < n; i++ {
		choice := choices[rand.Intn(len(choices))]
```

Reply:
Yep